From 11686426e6148dd55c250570f3d537a3b2c2724d Mon Sep 17 00:00:00 2001
From: Xin Ouyang <Xin.Ouyang@windriver.com>
Date: Mon, 23 Jul 2012 11:22:56 +0800
Subject: [PATCH] policycoreutils: fix format-security build error.

With this fix, we can build policycoreutils with
-Werror=format-security enabled.

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 sandbox/seunshare.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sandbox/seunshare.c b/sandbox/seunshare.c
index c493e98..d7eb0b9 100644
--- a/sandbox/seunshare.c
+++ b/sandbox/seunshare.c
@@ -311,8 +311,8 @@ static int seunshare_mount(const char *src, const char *dst, struct stat *src_st
  */
 static int sandbox_error(const char *string)
 {
-	fprintf(stderr, string);
-	syslog(LOG_AUTHPRIV | LOG_ALERT, string);
+	fprintf(stderr, "%s", string);
+	syslog(LOG_AUTHPRIV | LOG_ALERT, "%s", string);
 	exit(-1);
 }
 
-- 
1.7.5.4

